apps/api/src/app/topics/topics.controller.ts

```diff
@@ -46,7 +60,45 @@ export class TopicsController {
   }
 
   @ExternalApiAccessible()
-  @UseGuards(JwtAuthGuard)
+  @ApiQuery({
+    name: 'key',
+    type: String,
+    description: 'Topic key. Optional',
```

Comment:
no need to specify in the description `... Optional` as the `require` will generate a label for that

Reply:
I copy pasted and the source had them in our codebase. Something we need to review in other places.